README.md

```diff
@@ -66,9 +66,9 @@ See each command's linked README for more details, or click on [📚](https://do
 - `instrument`: Apply Datadog instrumentation to the given [AWS Lambda functions](src/commands/lambda).
 - `uninstrument`: Revert Datadog instrumentation from the given [AWS Lambda functions](src/commands/lambda).
 
-#### `metric`
+#### `measure`
 
-- Add [metrics](src/commands/metric) to a CI Visibility Pipeline trace or Job span in Datadog. [📚](https://docs.datadoghq.com/continuous_integration/pipelines/custom_tags_and_metrics/)
+- Add [measures](src/commands/measure) to a CI Visibility Pipeline trace or Job span in Datadog. [📚](https://docs.datadoghq.com/continuous_integration/pipelines/custom_tags_and_metrics/)
```

Comment:
is the idea to update the documentation too? We refer to metrics still. We might want to reflect that this link is not updated somewhere

Reply:
Yes, I'm currently changing the docs too (https://github.com/DataDog/documentation/pull/21897).
I will rename the mentions to the outdated docs as soon as that is merged.